### DISCUSSION TITLE
If press K (map to hover_doc) twice very fast, the hover window will not dismiss.

### DISCUSSION THREAD

**Discussion by towry** [nvimdev/lspsaga.nvim#933] [General]
I can switch to the hover window by `ctrl+w w`, and close it with `:q`.

the `require('lspsaga.hover'):has_hover()` returned false.

**Comment by glepnir** [comment]
if you map `K` with hover without `++keep` argument , press twice it will jump into the hover window. use `++keep` press k twice it will close the hover window.

**Reply by towry** [reply]
The behavior that I was expericing is if I press K too quick, it does not jump into the hover window, I have to add `nowait=true` option to my keybinding to make it work properly.

```
-- add nowait = true make jump to hover window works.
vim.keymap.set('n', 'K', 'Lspsaga hover_doc', { nowait = true })
```

Just for someone who would have similar issue.
